wai-extra/Network/Wai/Middleware/Autohead.hs

```diff
@@ -7,9 +7,7 @@ import Network.Wai
 import Data.Monoid (mempty)
 
 autohead :: Middleware
-autohead app req sendResponse
-    | requestMethod req == "HEAD" = app req { requestMethod = "GET" } $ \res -> do
-        let (s, hs, _) = responseToStream res
-        sendResponse $ responseBuilder s hs mempty
-    | otherwise = app req sendResponse
+autohead app req sendResponse = app req sendResponse
```

Comment:
This change is wrong; if wai-app-static doesn't require the autohead middleware, then it should be removed from https://github.com/yesodweb/wai/blob/master/wai-app-static/WaiAppStatic/CmdLine.hs#L103, not disabling the middleware itself, which is used in other places.